### DISCUSSION TITLE
directly export zod-openapi instance (`z`)

### DISCUSSION THREAD

**Discussion by scarf005** [asteasolutions/zod-to-openapi#182] [Ideas]
related: #175

It'd be convenient to directly import openapi-extended zod from `@anatine/zod-openapi`:
```ts
import { z } from "@anatine/zod-openapi"
```

over previous method:
```ts
import { z } from "zod"
import { extendZodWithOpenApi } from "@anatine/zod-openapi"

extendZodWithOpenApi(z)

export { z }
```

main benefits would be:
- slightly less boilerplate
- prevents mixing z from `zod` and `/path/extended-zod.ts`

**Comment by AGalabov** [comment]
@scarf005 just to point it out - you are using the wrong library as an example :/ 

Thank you for opening up this issue. However my personal feeling is not to implement this.

Yes, one can use:
```
import { z } from "@anatine/zod-openapi"
```

However:
1. I could "argue" that `import { z } from '@lib/zod'` (for example) is shorter and more up to the point and kind of represents that this is a locally patched version of zod. It also opens up a theoretical option to add more custom code there.
2. For "prevents mixing z from zod and /path/extended-zod.ts" - you can still mix the two quite easily if you want to?

I'll let @georgyangelov have a word as well.

**Reply by scarf005** [reply]
> you are using the wrong library as an example :/

oops, sorry.
